lib/less/importsPush.js

```diff
@@ -7,15 +7,16 @@
  *
  *
  * This file contains a modified version of the local function "imports.push" within the constructor of "less.Parser"
- * which has been taken from LESS v1.6.3 (https://github.com/less/less.js/blob/v1.6.3/lib/less/parser.js#L70-L111).
+ * which has been taken from LESS v1.6.3 (https://github.com/less/less.js/blob/v1.6.3/lib/less/parser.js#L70-L111)
+ * and applied commit https://github.com/less/less.js/commit/ccd8ebbfdfa300b6e748e8d7c12e3dbb0efd8371.
  * Modifications are marked with comments in the code.
  *
  */
 
 /* eslint-disable consistent-this, new-cap, no-invalid-this */
 "use strict";
 
-const less = require("less");
+const less = require("../thirdparty/less");
```

Comment:
This modification should be marked

Reply:
Actually there is not such a statement in the original code. As it's just an import to make the original code work I would expect that this is not relevant.